functions/collection.ts

```diff
@@ -0,0 +1,83 @@
+const AWS = require('aws-sdk');
+const isTest = process.env.JEST_WORKER_ID;
+const config = {
+  convertEmptyValues: true,
+  ...(isTest && {
+    endpoint: 'localhost:8000',
+    sslEnabled: false,
+    region: 'local-env',
+  }),
+};
+const docClient = new AWS.DynamoDB.DocumentClient(config);
+const table = (isTest) ? process.env.DYNAMO_DB_TEST_TABLE : process.env.DYNAMO_DB_GAME_TABLE;
+import { Game } from "./game";
+import { IDynamoObject } from "./gameManager";
+import { getCollection } from "./collectionManager";
+import { CollectionError } from "./collectionErrorHandler";
+
+export class Collection {
+    userID: string;
+    sortKey: string;    
+    collectionType: string;
```

Comment:
How would you use this attribute to extend the functionality of a wishlist? What if you want to add a few functions specifically for wishlists?